gtsam/slam/tests/testEssentialMatrixFactor.cpp

```diff
@@ -351,7 +353,117 @@ TEST (EssentialMatrixFactor3, minimization) {
   EXPECT_DOUBLES_EQUAL(0, graph.error(result), 1e-4);
 }
 
-} // namespace example1
+//*************************************************************************
+TEST(EssentialMatrixFactor4, factor) {
+  Key keyE(1);
+  Key keyK(1);
```

Comment:
Not sure how this test passed, shouldnt the keyK be initialized to 2?

Reply:
you are right.